domains/govin.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "My Portfolio",
+    "repo": "https://github.com/govindrajgupta/devfolio",
+    "owner": {
+        "username": "Govind Raj Gupta",
```

username does not match github username
```suggestion
        "username": "govindrajgupta",
```